CHANGELOG/CHANGELOG-1.1.md

```diff
@@ -0,0 +1,89 @@
+# Changelog 1.1
+
+## 1.1.0 - 2021-05-28
+
+### Added
+
+- Enable "noEncap" and "hybrid" traffic modes for clusters which include Windows Nodes. ([#2160](https://github.com/antrea-io/antrea/pull/2160) [#2161](https://github.com/antrea-io/antrea/pull/2161), [@lzhecheng] [@tnqn]) [Windows]
+  * Each Agent is responsible for annotating its Node resource with the MAC address of the uplink interface, using the "node.antrea.io/mac-address" annotation; the annotation is used to forward Pod traffic
+- Add a generic mechanism to define policy rules enforced on all the network endpoints belonging to the same Namespace as the target of the AppliedTo; this makes it very easy to define an Antrea CNP to allow same-Namespace traffic across all Namespaces. ([#1961](https://github.com/antrea-io/antrea/pull/1961), [@Dyanngg])
```

Comment:
```suggestion
- Add a generic mechanism to define policy rules enforced on all the network endpoints belonging to the same Namespace as the target of the AppliedTo; this makes it very easy to define an Antrea CNP to only allow same-Namespace traffic (Namespace isolation), across all Namespaces in the cluster or for select Namespaces. ([#1961](https://github.com/antrea-io/antrea/pull/1961), [@Dyanngg])
```

Reply:
> an Antrea CNP to only allow same-Namespace traffic

don't you need 2 CNPs to do that? one for allowing traffic and one for default-deny?

Reply:
or I guess 1 CNP with 2 rules would do that as well

Reply:
Yes the example CNP is one with two ordered rules, allow same Namespace at the top and default-deny at bottom

Reply:
Ok I'll incorporate your suggestion before merging